server/controllers/entities/lib/add_relatives.coffee

```diff
@@ -0,0 +1,41 @@
+# Enrich ../by_uris results with entities related to the directly
+# requested entities, following those entities claims
+
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+getEntitiesByUris = require './get_entities_by_uris'
```

Comment:
`add_relatives` file has `get_entities_by_uris` dependency, while it is called right after [get_entities_by_uris](https://github.com/inventaire/inventaire/pull/134/files#diff-e5405d93d05274b0a584fb5fcadd52cbR38). Wouldnt make it easier if `add_relatives` is within `get entities by uris` ?